src/js/modules/articles/selectors.js

```diff
@@ -79,6 +90,52 @@ export const getArticlesByContributor = createSelector(
 );
 
 /**
+ * The selector returns an object that contains all articles an illustrator has
+ *   illustrated for (featured media only).
+ */
+export const getIllustratorArticles = createSelector(
+  [ getIllustratorIllustrations, getArticlesWithContributors ],
```

Comment:
That's...not a great name either. I see why you chose it, but it's a little hard to read